patches/Fix_escaping_of_arguments_passed_to_LaunchProcess()_in_JavaBinaryLauncher__CreateClasspath.patch

```diff
@@ -0,0 +1,21 @@
+Subject: [PATCH] Fix escaping of arguments passed to LaunchProcess() in JavaBinaryLauncher::CreateClasspathJar on Windows
```

Comment:
Please rename to `patches/0005-Fix_escaping_of_arguments_passed_to_LaunchProcess()_in_JavaBinaryLauncher__CreateClasspath.patch`.

Reply:
Done